src/components/Layout/Table/index.js

```diff
@@ -43,17 +44,38 @@ export const TH = (props) => (
   </th>
 )
 
-export const TD = (props) => (
-  <td
-    className={cx(
-      'govuk-table__cell',
-      props.width ? `govuk-!-width-${props.width}` : null,
-      props.padding ? `govuk-!-padding-${props.padding}` : null,
-      props.type ? `govuk-table__cell--${props.type}` : null,
-      props.border ? `border-${props.border}` : null,
-      props.className
-    )}
-  >
-    {props.children}
-  </td>
-)
+export const TD = (props) => {
+  return props.className && props.className.includes('description') ? (
+    <>
+      <td
+        className={cx(
+          'govuk-table__cell',
+          props.width ? `govuk-!-width-${props.width}` : null,
+          props.padding ? `govuk-!-padding-${props.padding}` : null,
+          props.type ? `govuk-table__cell--${props.type}` : null,
+          props.border ? `border-${props.border}` : null,
+          props.className
+        )}
+      >
+        <TruncateText
+          text={props.children}
+          numberOfLines="5"
+          linkClassName={props.className}
+        ></TruncateText>
+      </td>
+    </>
+  ) : (
+    <td
+      className={cx(
+        'govuk-table__cell',
+        props.width ? `govuk-!-width-${props.width}` : null,
+        props.padding ? `govuk-!-padding-${props.padding}` : null,
+        props.type ? `govuk-table__cell--${props.type}` : null,
+        props.border ? `border-${props.border}` : null,
+        props.className
+      )}
+    >
+      {props.children}
+    </td>
+  )
+}
```

Comment:
Now that we have our `<TruncateText >` component we should be able to make the parent responsible for passing that into the `<TD>`, which would leave our original `<TD>` untouched, remove the duplicate code here, and make the parent code easier to read. This should work because the `<TD>` just renders whatever children are passed into it using [`props.children`](https://reactjs.org/docs/composition-vs-inheritance.html#containment).

https://github.com/LBHackney-IT/repairs-hub-frontend/blob/a803af6d0cba0474ea526e88b59d7fd4906bc20b/src/components/WorkOrders/WorkOrderRow.js#L35

should be able to be changed to

```
<TD className="description table-font-size"><TruncateText text={description} numberOfLines="5" /></TD>
```
And then we should be able to revert all changes to `<TD>` in this file.